vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/batchRegistration/GeneralInformationLayout.java

```diff
@@ -0,0 +1,153 @@
+import com.vaadin.flow.data.renderer.ComponentRenderer;
+import com.vaadin.flow.theme.lumo.LumoUtility.IconSize;
+import com.vaadin.flow.theme.lumo.LumoUtility.Margin.Left;
+import java.util.ArrayList;
+import java.util.List;
+
+
+/**
+ * <b>General Information Layout </b>
+ * <p>
+ * Layout in which the user will provide the general information necessary during sample
+ * registration
+ * </p>
+ */
+class GeneralInformationLayout extends VerticalLayout {
+
+  public final TextField batchNameField = new TextField("Batch Name");
+  public final RadioButtonGroup<MetaDataTypes> dataTypeSelection = new RadioButtonGroup<>();
+  public final Button cancelButton = new Button("Cancel");
+  public final Button nextButton = new Button("Next");
+  private final GeneralInformationLayoutHandler generalInformationLayoutHandler;
+
+  public GeneralInformationLayout() {
+    initContent();
+    this.setSizeFull();
+    generalInformationLayoutHandler = new GeneralInformationLayoutHandler();
+  }
+
+  private void initContent() {
+    Span generalInformationHeader = new Span("General Information");
+    generalInformationHeader.addClassNames("text-xl", "font-bold", "text-secondary");
+    add(generalInformationHeader);
+    initBatchLayout();
+    initDataTypeLayout();
+    initButtonLayout();
+  }
+
+  private void initBatchLayout() {
+    HorizontalLayout batchLayout = new HorizontalLayout();
+    batchLayout.add(batchNameField);
+    add(batchLayout);
+  }
+
+  private void initDataTypeLayout() {
+    VerticalLayout dataTypeLayout = new VerticalLayout();
+    dataTypeLayout.setMargin(false);
+    dataTypeLayout.setPadding(false);
+    Span dataTypeHeader = new Span("Type of Data");
+    dataTypeHeader.addClassNames("text-l", "font-bold", "text-secondary");
+    dataTypeLayout.add(dataTypeHeader);
+    Span dataTypeDescription = new Span(
+        "There is a minimum amount of information required. All samples must conform the expected metadata values. The most suitable checklist for sample registration depends on the type of the sample.");
+    dataTypeLayout.add(dataTypeDescription);
+    initDataTypeSelection();
+    dataTypeLayout.add(dataTypeSelection);
+    dataTypeSelection.setClassName(Left.MEDIUM);
+    dataTypeLayout.setSizeFull();
+    add(dataTypeLayout);
+  }
+
+  private void initDataTypeSelection() {
+    dataTypeSelection.setItems(MetaDataTypes.values());
+    dataTypeSelection.setValue(dataTypeSelection.getListDataView().getItem(0));
+    dataTypeSelection.addThemeVariants(RadioGroupVariant.LUMO_VERTICAL);
+    dataTypeSelection.setRenderer(new ComponentRenderer<>(MetaDataTypes -> {
+      Span metaDataType = new Span(MetaDataTypes.metaDataType);
+      Icon infoIcon = new Icon(VaadinIcon.INFO_CIRCLE);
+      infoIcon.addClassNames(IconSize.SMALL);
+      infoIcon.setColor("#77828f");
+      infoIcon.setTooltipText(MetaDataTypes.metaDataDescription);
+      return new HorizontalLayout(metaDataType, infoIcon);
+    }));
+  }
```

Comment:
NIT: I don't like that the tooltip is not aligned, but next to the words.

Reply:
tooltip icon*

Reply:
Good point i'llm ake a note of it and fix it during the next iteration 👀